hera_mc/geo_handling.py

```diff
@@ -127,23 +127,27 @@ def get_station_types(self, add_stations=True):
         add_stations:  if True, add all of the stations to their types
                        if False, just return station types
```

Comment:
Again, this function no longer returns anything.

Also, it might be helpful to add an example of when one might want to specify `add_stations = False`. It would simplify this function and some of its callers if the stations were always loaded up (i.e. if the `add_stations` argument were removed).

Reply:
Thought about this from a performance point of view, then decided to do it since minimal hit and, as pointed out, simpler.